client/app/routes/collection/add.js

```diff
@@ -1,4 +1,124 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+
+    model() {
+        return {
+            submission_form_name: 'Preprints Submission Form',
+            sections: [
+                'upload',
+                'disciplines',
+                'basic info',
+                'authors',
+                'submit'
+            ],
+            initial_state: {
+                upload_section: {
+                    state: ['unsaved', 'editing'],
+                    value: undefined
+                },
+                disciplines_section: {
+                    state: ['disabled'],
+                    value: undefined
+                },
+                basic_info_section: {
+                    state: ['disabled'],
+                    value: undefined
+                },
+                authors_section: {
+                    state:  ['disabled'],
+                    value: undefined
+                },
+                submit_button: {
+                    state: ['disabled'],
+                    value: undefined
+                },
+                preprint_file_upload_widget: {
+                    state: ['undefined'],
+                    value: undefined
+                },
+                preprint_title_widget: {
+                    state: ['undefined'],
+                    value: undefined
+                },
+                save_upload_section_widget: {
+                    state: ['undefined'],
+                    value: undefined
+                }
+            },
+            actions: [{
+                type: 'create_widget',
+                parameters: {
+                    widget_component: 'file-uploader',
+                    description: 'Choose the preprint file to upload',
+                    section: 'upload',
+                    output: 'preprint_file_data'
+                },
+                output: 'preprint_file_upload_widget',
+                conditions: [{
+                    all: [{
+                        parameter: 'upload_section',
+                        state: 'unsaved',
+                    }, {
+                        parameter: 'preprint_file_upload_widget',
```

Comment:
Yes it should. It means we need to decide on what the difference in meaning for the parameter to be wholly absent from the state, or that it's defined and its value is literally `undefined`.